.github/workflows/build.yml

```diff
@@ -38,4 +38,5 @@ jobs:
         uses: actions/upload-pages-artifact@v3
 
       - name: Deploy to GitHub Pages
+        continue-on-error: ${{vars.IS_CANONICAL_DOCS_REPO != 'true'}}
```

Comment:
I guess this makes sense if we start doing more processing on GitHub actions, but I would probably hold off on it until #90 is resolved, for all the reasons you outlined there.

I'd been running all my tests over local builds and it works okay so far.

Reply:
Not sure I follow, this doesn't actually change anything. The website won't actually deploy unless the owner of the fork explicitly enables it.

Reply:
Yeah, forgot about that and agree this is enough protection against accidental proliferation.